src/main/java/com/yelp/nrtsearch/server/luceneserver/search/collectors/additional/TermsCollectorManager.java

```diff
@@ -198,7 +198,7 @@ public static TermsCollectorManager buildManager(
   protected TermsCollectorManager(
       String name,
       int size,
-      Map<String, Supplier<AdditionalCollectorManager<? extends Collector, CollectorResult>>>
+      Map<String, Supplier<AdditionalCollectorManager<?, CollectorResult>>>
```

Comment:
Hi @aprudhomme I am getting this error if I don't do this change after making UNSET_VALUE configurable. Seems a bit unrelated coz I put a debugger point and it does not seem to be in the flow. 

`/src/main/java/com/yelp/nrtsearch/server/luceneserver/search/collectors/additional/TermsCollectorManager.java:209: error: incompatible types: Map<String,Supplier<AdditionalCollectorManager<? extends Collector,CollectorResult>>> cannot be converted to Map<String,Supplier<AdditionalCollectorManager<?,CollectorResult>>>
: new NestedCollectorManagers(nestedCollectorSuppliers);`